### DISCUSSION TITLE
Reversed Controls for Fireplace

### DISCUSSION THREAD

**Discussion by pavelgigov** [aarons22/homebridge-bond#175] [General]
Hi,

I was able to instal and configure the plug in for Homebridge, but something super weird happens. Everything is reversed in HomeKit. E.g. I have to ask Siri to "Turn the fireplace on" to turn it off, and vice-versa, I have to ask Siri to "Turn the fireplace off", to turn it on. 

Any clues?

**Comment by chrismerck** [comment]
Is it also reversed in the Bond Home app? You may have a fireplace with a toggle signal, in which case the Bond Bridge doesn't really know if it the device is on or off.

**Comment by pavelgigov** [comment]
No, it works and indicates normally in Bond. and when I use Google howe to control it, the voice commands match too.

**Comment by StarChild71** [comment]
@pavelgigov I was able to address this by recording the commands in Bond as a Device instead of a Fireplace. Now the on/off works as expected in HomeKit.
